### DISCUSSION TITLE
Do you have a top level "latest" in releases?

### DISCUSSION THREAD

**Discussion by bszerdy** [flipt-io/flipt#2878] [General]
for reasons, I need to pull the binary and make a local container. I retrieve the binary via a `wget` but I currently need to target the version. Can you alias your current releases to something like: `https://github.com/flipt-io/flipt/releases/download/latest` ? Thanks

**Comment by markphelps** [comment]
Hey @bszerdy !

We use Go Releaser to actually [publish the release to GitHub](https://goreleaser.com/customization/release/#github). I'll see if Go Releaser supports this natively, or if we can modify our release process to always also publish a `latest` release to GitHub.

In the meantime, we currently use this trick to get the latest release from our install script: https://github.com/flipt-io/flipt/blob/908b18879172d40bab2013eda05bbcad9f73788b/install.sh#L22-L25

Does that help unblock you?

**Reply by bszerdy** [reply]
Yes, thank you!
